lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -162,6 +213,7 @@ public EvaluatorRequest build() {
      * @param rd the descriptor used to pre-fill this request.
      * @return this
      */
+    @Deprecated
```

Comment:
Please add javadoc `@deprecated` as well and document when it was deprecated and what to use instead.